src/main/kotlin/com/dansplugins/factionsystem/listener/PlayerInteractListener.kt

```diff
@@ -154,7 +154,14 @@ class PlayerInteractListener(private val plugin: MedievalFactions) : Listener {
             }
         }
         val claimService = plugin.services.claimService
-        val claim = claimService.getClaim(clickedBlock.chunk) ?: return
+        val claim = claimService.getClaim(clickedBlock.chunk)
+        if (claim == null) {
+            if (plugin.config.getBoolean("wilderness.preventBlockInteraction", false)) {
+                event.isCancelled = true
+                event.player.sendMessage("$RED${plugin.language["CannotInteractBlockInWilderness"]}")
```

Comment:
This can show up a lot and spam the screen, might be good to make these alerts configurable.

Reply:
Addressed